src/amo/components/Page/index.js

```diff
@@ -77,7 +80,9 @@ export const PageBase = ({
 
   return (
     <div className="Page-amo">
-      {showVPNPromo ? <VPNPromoBanner /> : null}
+      {showVPNPromo && _config.get('enableFeatureVPNPromo') && (
```

Comment:
I guess that's now needed here because we need to disable the feature as well as the experiment and the feature flag in `<VPNPromoBanner />` wouldn't work?

Reply:
Yes, I want to decouple disabling the feature from disabling the experiment. There are edge cases where it matters.

I think it makes the most sense to have the flag here. It's temporary (as all feature flags are), and it is designed to disable anything to do with the banner. Adding it here means we don't have to worry about it inside the `VPNPromoBanner` because that component will never even render if the flag is false. It makes testing the flag simpler, and requires fewer tests.

It also means that the changes required when we do want to remove the flag are fewer and simpler as it's all right here, and we don't need to touch the `VPNPromoBanner` at all when it's time to remove the flag.